client/core/notification.go

```diff
@@ -100,19 +100,19 @@ type Notification interface {
 // FeePaymentNote is a notification regarding registration fee payment.
 type FeePaymentNote struct {
 	db.Notification
-	ConfirmationsRequired uint32 `json:"confirmationsrequired,omitempty"`
-	Confirmations         uint32 `json:"confirmations,omitempty"`
+	Confirmations uint32 `json:"confirmations,omitempty"`
+	DexURL        string `json:"dexurl,omitempty"`
```

Comment:
See #374, but use just `DEX` and `json:"dex"` here for now. DEX field should never be empty. `omitempty` is also confusing for `Confirmations`, since `0` is a valid value. Maybe a `*uint32` instead?

Reply:
Makes sense :)